.babelrc

```diff
@@ -1,5 +1,5 @@
 {
-  "presets": ["flow", "es2015", "react", "stage-0"],
+  "presets": ["flow", "env", "react", "stage-0"],
```

Comment:
we should remove stage-0

Reply:
To remove `stage-0`, probably you will need have add some babel plugins.

Reply:
yep, let's tranpile less

Reply:
With `env`, you have more possibilities for transpile less.

Ex:

```JSON
{
  "presets": ["flow", [
    "env",
    {
      "loose": true,
      "exclude": [
        "transform-es2015-typeof-symbol"
      ],
      "targets": {
        "browsers": [
          "last 2 versions",
          "IE >= 9"
        ]
      }
    }
  ], "react", "stage-0"],
  "plugins": [
    ["relay", {
      "schema": "test/fixtures/graphql/schema.json"
    }]
  ]
}
```

Reply:
which browsers should we support? the same as relay?

Reply:
i guess so

Reply:
It would be safest, but if you want to transpile less, you'll have add browsers with browserlist https://github.com/ai/browserslist

Reply:
Learn about `loose mode` you can gain a few kb
http://2ality.com/2015/12/babel6-loose-mode.html

```JSON
"presets": ["flow", ["env", { "loose": true }], "react", "stage-0"]
```

Reply:
I think we should merge this and improve browsers support in another PR

Reply:
Can you resolve this https://github.com/releasy/react-releasy/issues/4 in this PR too?

Reply:
You use generator functions or async ?